velox/exec/Spiller.cpp

```diff
@@ -360,9 +364,10 @@ void Spiller::ensureSorted(SpillRun& run) {
         });
     run.sorted = true;
   }
-  if (sortTimeUs != 0) {
-    updateSpillSortTime(sortTimeUs);
+  if (sortTimeUs == 0) {
+    sortTimeUs = 1;
   }
+  updateSpillSortTime(sortTimeUs);
```

Comment:
nit: 
```
// NOTE: always set a non-zero sort time to avoid flaky test which check on sort time.
updateSpillSortTime(std::max(1, sortTimeUs));
```